bin/sgcloud

```diff
@@ -29,6 +29,7 @@
   }
 
   caporal.version(pkg.version)
+  process.env['SDK_BETA'] = process.env['SDK_BETA'] || semver.prerelease(pkg.version).includes('beta')
```

Comment:
A non beta version of the SDK will crash here.
`semver.prerelease(pkg.version)` will return `null` when its not a prerelease